### ISSUE TITLE
Composite primary key support

### ISSUE THREAD

**Issue by caseyandgina** [brendon/acts_as_list#417]
Can you please add compatibility with https://github.com/composite-primary-keys/composite_primary_keys ?

Currently, this is an example of SQL that ends up being generated when attempting to use the acts_as_list gem with a table with a composite primary key:

```
UPDATE "feeds"."account_feeds" SET "position" = ("feeds"."account_feeds"."position" + 1) WHERE (1 = 1) AND ("feeds"."account_feeds".account_id,feed_id != 110896047323649606,158) AND ("feeds"."account_feeds"."position" >= 1)
```

**Comment by brendon** [comment]
Hi @caseyandgina, I think I've seen Rails integrating composite keys into AR itself in version 7 lately?

Would you be interested in doing up a PR that brings this into Acts As List? I'm a bit short of time to do it myself but an very happy to look at PR's and provide feedback etc... :)

**Comment by brendon** [comment]
Hi @caseyandgina, there's an open PR to add this functionality. Just waiting for @caseyandgina to finalise the tests.

**Comment by brendon** [comment]
This has been implemented and released as `1.2.0`.
